src/apis/forecast-api-service.ts

```diff
@@ -1,103 +1,103 @@
-import { Allocation } from "./schemas/allocation";
-import { Project } from "./schemas/project";
-import { Task } from "./schemas/task";
+import { Allocation } from "./schemas/forecast/allocation";
+import { Project } from "./schemas/forecast/project";
+import { Task } from "./schemas/forecast/task";
 import fetch from "node-fetch";
 
 export interface ForecastApiService {
-    getAllocations: () => Promise<Allocation[]>;
-    getProjects: () => Promise<Project[]>;
-    getProject: (projectId: string) => Promise<Project>;
-    getTasksByProject: (projectId: number) => Promise<Task[]>;
-    getAllTasks: () => Promise<Task[]>;
-    getProjectSprints: (projectId: number) => Promise<Sprint[]>;
-    getTimeEntriesByProject: (projectId: number) => Promise<TimeEntry[]>;
+	getAllocations: () => Promise<Allocation[]>;
```

Comment:
Spaces are set to 4 they should be 2 as said in a previous comment

Reply:
Actually this seems fine in the VSCode, perhaps its just the github UI not showing indentation correctly. If it seems to you the indentation is set to 2 spaces then this is fine.